[2024-11-06T15:14:30.357+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-06T15:14:30.565+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.create_postgres_table scheduled__2024-11-03T00:00:00+00:00 [queued]>
[2024-11-06T15:14:30.629+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v01.create_postgres_table scheduled__2024-11-03T00:00:00+00:00 [queued]>
[2024-11-06T15:14:30.636+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-11-06T15:14:30.791+0000] {taskinstance.py:2888} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2024-11-03 00:00:00+00:00
[2024-11-06T15:14:31.120+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=201) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-06T15:14:31.052+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v01', 'create_postgres_table', 'scheduled__2024-11-03T00:00:00+00:00', '--job-id', '179', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres.py', '--cfg-path', '/tmp/tmpka9kmbxy']
[2024-11-06T15:14:31.142+0000] {standard_task_runner.py:72} INFO - Started process 216 to run task
[2024-11-06T15:14:31.142+0000] {standard_task_runner.py:105} INFO - Job 179: Subtask create_postgres_table
[2024-11-06T15:14:31.616+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_postgres_operator_v01.create_postgres_table scheduled__2024-11-03T00:00:00+00:00 [running]> on host b0b6b56c3157
[2024-11-06T15:14:33.866+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='johnwell' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v01' AIRFLOW_CTX_TASK_ID='create_postgres_table' AIRFLOW_CTX_EXECUTION_DATE='2024-11-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-03T00:00:00+00:00'
[2024-11-06T15:14:33.868+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-06T15:14:34.416+0000] {sql.py:266} INFO - Executing: 
        create table if not exists dag_runs (
            dt date,
            dag_id character varying,
            primary key (dt, dag_id)
        )
        
[2024-11-06T15:14:34.561+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-11-06T15:14:34.710+0000] {base.py:84} INFO - Retrieving connection 'postgres_localhost'
[2024-11-06T15:14:34.981+0000] {sql.py:509} INFO - Running statement: 
        create table if not exists dag_runs (
            dt date,
            dag_id character varying,
            primary key (dt, dag_id)
        )
        , parameters: None
[2024-11-06T15:14:35.020+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-06T15:14:35.020+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v01, task_id=create_postgres_table, run_id=scheduled__2024-11-03T00:00:00+00:00, execution_date=20241103T000000, start_date=20241106T151430, end_date=20241106T151435
[2024-11-06T15:14:35.508+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-06T15:14:35.770+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-06T15:14:35.785+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2024-11-06T15:14:36.169+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
